datastore/db/bytebase/FertiScan/schema_0.0.9.sql

```diff
@@ -0,0 +1,252 @@
+--Schema creation "fertiscan_0.0.9"
+DO
+$do$
+BEGIN
+IF (EXISTS (SELECT 1 FROM information_schema.schemata WHERE schema_name = 'fertiscan_0.0.9')) THEN
+
+
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+    CREATE TABLE "fertiscan_0.0.9"."users" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "email" text NOT NULL UNIQUE,
+    "registration_date" timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" timestamp
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."picture_set" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture_set" json NOT NULL,
+    "owner_id" uuid NOT NULL REFERENCES "fertiscan_0.0.9".users(id),
+    "upload_date" date NOT NULL DEFAULT current_timestamp,
+    "name" text
+    );
+        
+    alter table "fertiscan_0.0.9".users ADD "default_set_id" uuid REFERENCES "fertiscan_0.0.9".picture_set(id);
+
+    CREATE TABLE "fertiscan_0.0.9"."picture" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture" json NOT NULL,
+    "nb_obj" int,
+    "picture_set_id" uuid NOT NULL REFERENCES "fertiscan_0.0.9".picture_set(id),
+    "verified" boolean NOT NULL DEFAULT false,
+    "upload_date" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."province" (
+    "id" SERIAL PRIMARY KEY,
+    "name" text UNIQUE NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."region" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "province_id" int REFERENCES "fertiscan_0.0.9".province(id),
+    "name" text NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."location" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text,
+    "address" text NOT NULL,
+    "region_id" uuid References "fertiscan_0.0.9".region(id)
+    );    
+    
+    CREATE TABLE "fertiscan_0.0.9"."organization_information" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "name" text NOT NULL,
+        "website" text,
+        "phone_number" text,
+        "location_id" uuid REFERENCES "fertiscan_0.0.9".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.9"."organization" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text NOT NULL,
```

Comment:
Is the `name` field still needed in organization?

Reply:
Will be fixed in https://github.com/ai-cfia/ailab-datastore/issues/95. Resolving this.